applet/src/radiostate.h

```diff
@@ -21,9 +21,9 @@ extern int rst_hdr_sap ;
 extern int rst_hdr_src ;
 extern int rst_hdr_dst ;
 
-void rst_voice_lc_header(int src, int dst);
+void rst_voice_lc_header(int src, int dst, int groupcall);
```

Comment:
how about passing the pointer to the complete packet, and move the packet interpretation to the rst_ methods?